### DISCUSSION TITLE
Team Selection

### DISCUSSION THREAD

**Discussion by andrewhuffaker** [nbrown02/Flow-Metrics-Epics-Features#1] [General]
Hi Nick, asking as a simpleton....
Why doesn't this report have team level filters like the FlowViz reports do?

**Comment by nbrown02** [comment]
Hey @andrewhuffaker - tbh it was just for simplicity reasons (me being lazy) with sharing something i've already built and also to speed up data load.

A version that has all teams in a project could be built, just finding the time :)
